src/menu/DatasetCard.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import {Card, Nav, Tab, Row} from 'react-bootstrap';
+import {INamedSet, ENamedSetType, RestBaseUtils, RestStorageUtils} from 'tdp_core';
+import {NamedSetList, useAsync, IStartMenuDatasetDesc} from 'ordino';
+import {UserSession} from 'phovea_core';
+import {DatasetSearchBox} from './DatasetSearchBox';
+
+
+
+
+export default function DatasetCard({name, headerIcon, tabs, viewId, datasource}: IStartMenuDatasetDesc) {
```

Comment:
The interface `IStartMenuDatasetDesc` is defined in ordino with `datasource: any`. I'd suggest to remove the `datasource` from the interface and create a new local interface here that extends `IStartMenuDatasetDesc`. Then you can set the correct typing for the `datasource`.

Reply:
Furthermore, please rename `datasource` to `dataSource` everywhere, since the origin is "data source".